{"name": "Get orders for authorized user", "status": "failed", "statusDetails": {"message": "AssertionError: assert 403 == 200\n +  where 403 = <Response [403]>.status_code", "trace": "self = <Diplom_2.tests.test_get_user_orders.TestGetUserOrders object at 0x00000128F9425D10>\n\n    @allure.title('Get orders for authorized user')\n    def test_get_orders_authorized(self):\n        response = self.api.get_user_orders(self.token)\n>       assert response.status_code == 200\nE       assert 403 == 200\nE        +  where 403 = <Response [403]>.status_code\n\ntests\\test_get_user_orders.py:22: AssertionError"}, "start": 1751986480944, "stop": 1751986481138, "uuid": "c42010b8-5e47-4c74-966d-b7ad844edc48", "historyId": "c61394d2ac4d3f409a6b251ffd517580", "testCaseId": "c61394d2ac4d3f409a6b251ffd517580", "fullName": "tests.test_get_user_orders.TestGetUserOrders#test_get_orders_authorized", "labels": [{"name": "feature", "value": "Get User Orders"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_get_user_orders"}, {"name": "subSuite", "value": "TestGetUserOrders"}, {"name": "host", "value": "DESKTOP-42BDC19"}, {"name": "thread", "value": "24480-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_get_user_orders"}]}